### PR TITLE
Descriptions added

### PR DESCRIPTION
**Pull request by JKtranslator**
Descriptions added for the cards

### CODE REVIEW

**Review by Majorfi — APPROVED** [review]
LGTM, I will let @pentcle check it too!

File: utils/yearnApps.tsx [hunk 1]
```diff
@@ -3,126 +3,126 @@ const YEARN_APPS = [
...
 		'title': 'faq',
 		'url': 'https://docs.yearn.finance/resources/faq',
-		'description': '',
+		'description': 'Written guides and explanations for the most common questions about Yearn.',
 		'icon': '/yearnApps/docs.favicon.svg'
 	},
 	{
 		'id': 'vaults',
 		'title': 'vaults',
 		'url': 'yearn.finance',
```
Comment (pentcle) [inline utils/yearnApps.tsx:19]: https://yearn.finance/#/portfolio

Comment (pentcle) [inline utils/yearnApps.tsx]: https://yearn.finance/#/portfolio

File: utils/yearnApps.tsx [hunk 2]
```diff
@@ -3,126 +3,126 @@ const YEARN_APPS = [
...
 		'title': 'Vaults at Yearn',
 		'url': 'https://vaults.yearn.finance/',
-		'description': '',
+		'description': 'List of all vaults and their strategies, with a short explanation of what each strategy does.',
 		'icon': '/yearnApps/docs.favicon.svg'
 	},
 	{
 		'id': 'Yearn watch',
 		'title': 'Yearn watch',
 		'url': 'yearn.watch',
```
Comment (pentcle) [inline utils/yearnApps.tsx]: https://yearn.watch/

**Comment by pentcle** [comment]
@JKtranslator thanks for this!
2 of the links need the https, then good to go

**Review by pentcle — APPROVED** [review]

**Comment by pentcle** [comment]
i updated the 2 urls that were missing https://
ready for @Majorfi to auth vercel

**Pull request merged**